README.md

```diff
@@ -51,6 +58,8 @@ If you are on a `SNAPSHOT` version, you can force redownloads of modules/depende
  ./gradlew[.bat] clean build --parallel --refresh-dependencies
 ```
 
+Some options are in 'gradle.properties', and you add your own in order to improve gradle speed
```

Comment:
you need to be a bit more explicit. what options? and what is expected to be modified? are those really things that cas should document or a gradle-person should already know?